_config.yaml

```
@@ -1,29 +1,20 @@
-title: Data Lab Training
-training_title: MONTH-TOPIC
+title: ALSF Data Lab Training
+training_title: 2023 May Single-cell RNA-Seq Workshop
 # The name of the repository (e.g., 2020-may-training)
 # We use training-specific-template to help with development
-repository: training-specific-template
+repository: 2023-may-training
 # Full URL to the repository
 # Again, using training-specific-template to aid in development
-repository_url: https://github.com/AlexsLemonade/training-specific-template
-start_date: YYYY-MM-DD
-end_date: YYYY-MM-DD
+repository_url: https://github.com/AlexsLemonade/2023-may-training
+start_date: 2023-05-15
+end_date: 2023-05-19
 # Specific release in AlexsLemonade/training-modules
 # We use master to make development easier
-release_tag: master
+release_tag: 2023-may
```

Usually we leave this change until the release exists. Then links will work in testing. (Note that this also means that this PR should _not_ close #4, but just gets most of the way there.
```suggestion
release_tag: master
```